src/domain/page/Page.tsx

```diff
@@ -5,8 +5,6 @@ import { useTranslation } from 'react-i18next';
 import Layout from '../../common/layout/Layout';
 import Sidebar from '../../common/sidebar/Sidebar';
 import Expandable from '../../common/expandable/Expandable';
-import InternalLink from '../../common/internalLink/InternalLink';
-import Text from '../../common/text/Text';
 import Button from '../../common/button/Button';
```

Comment:
TBH, I tried but turned out to be a bigger task than I estimated, so I left it out of the scope of this PR since we have a [JIRA ticket](https://helsinkisolutionoffice.atlassian.net/browse/VEN-431) for that.